libs/sync/src/lib/upload/sync.component.ts

```diff
@@ -27,10 +28,16 @@ export class SyncComponent implements OnInit, OnDestroy {
     private router: Router,
     private toast: SisToastService,
     private cdr: ChangeDetectorRef,
+    private sdk: SDKService,
   ) {}
 
   ngOnInit() {
-    this.syncService.setCurrentSourceId(location.pathname.split('/upload/sync/')[1]?.split('/')[0]);
+    this.syncService.setCurrentSourceId(
+      (this.sdk.nuclia.options.standalone ? location.hash : location.pathname)
+        .split('/upload/sync/')[1]
+        ?.split('/')[0]
+        .split('?')[0] || '',
```

Comment:
this code is hard to read and might be are to debug in case of problem later. I don't really understand why you use global location instead of angular’s ActivatedRoute, it seems like it would be safer and clearer to get the source Id from there.